### PR TITLE
Updates to readme  to keep current

### PR DESCRIPTION
**Pull request by kcoriordan**
Signed-off-by: kcoriordan <koriordan@vmware.com>

## What this PR does / why we need it
- Removes duplicate doc link under getting started
- Adds link to roadmap 
- Adds kpack to the packages list
- Makes list of packages alphabetical
- Updates the install command for accuracy/consistency with other doc

## Details for the Release Notes (PLEASE PROVIDE)

```release-note

```

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,8 +66,8 @@ choco install tanzu-community-edition
     * `tar zxvf <release tarball>` on Mac/Linux.
 1. Enter the directory of the unpacked release.
 1. Run the install script.
-    * `install.bat` on Windows as Administrator.
-    * `install.sh` on Mac/Linux
+    * `./install.bat` on Windows as Administrator.
```
Comment (stmcginnis) [inline README.md:69]: Windows should be `.\`

File: README.md [hunk 2]
```diff
@@ -77,25 +76,27 @@ platform. Packages included, by default, in Tanzu Community Edition are:
...
 | Velero | Provides disaster recovery capabilities | [Velero package docs](./addons/packages/velero) |
-| Multus CNI | Provides ability for attaching multiple network interfaces to pods in Kubernetes | [Multus CNI package docs](./addons/packages/multus-cni) |
 | Whereabouts | Provides A CNI IPAM plugin that assigns IP addresses cluster-wide | [Whereabouts package docs](./addons/packages/whereabouts) |
-| Sriov Network Device Plugin | The SR-IOV Network Device Plugin is Kubernetes device plugin for discovering and advertising SR-IOV virtual functions (VFs) available on a Kubernetes host. | [Sriov Network Device Plugin package docs](./addons/packages/sriov-network-device-plugin) |
 
 ## Contributing
 
 If you are ready to jump in and test, add code, or help with documentation,
-follow the instructions on our [Contribution Guidelines](./CONTRIBUTING.md) to
+follow the instructions on our [Contribution Guidelines](https://tanzucommunityedition.io/docs/latest/contribute/contributing/) to
```
Comment (stmcginnis) [inline README.md:99]: Shouldn't this be a relative link?

Comment (kcoriordan) [reply README.md:99]: If I want the contributing file to open in the docs site - I thought no?
Can I do a relative doc link when the readme is outside the Hugo root `/site`.

Comment (stmcginnis) [reply README.md:99]: Ah, sorry. You are right with the way you have this. I misunderstood what was going on.

**Review by stmcginnis — APPROVED** [review]
Looks great!

**Pull request merged**